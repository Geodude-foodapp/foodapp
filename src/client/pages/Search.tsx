import React from 'react';

export default () => (
  <section id='search'>
    <h1>Search Page</h1>
  </section>
);
