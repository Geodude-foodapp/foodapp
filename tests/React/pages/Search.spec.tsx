import { render } from '@testing-library/react';
import React from 'react';
import Search from '../../../src/client/pages/Search';

describe('Sign Up Page', () => {
  beforeAll(() => render(<Search />));
  it('matches the snapshot', () => {
    const searchPage = document.querySelector('#search');
    expect(searchPage).toMatchSnapshot();
  });
});
