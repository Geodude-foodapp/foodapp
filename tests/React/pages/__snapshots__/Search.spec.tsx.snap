// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up Page matches the snapshot 1`] = `
<section
  id="search"
>
  <h1>
    Search Page
  </h1>
</section>
`;
